snuba/cli/bootstrap.py

```diff
@@ -69,7 +69,7 @@ def bootstrap(
         topics = {}
         for name in ACTIVE_DATASET_NAMES:
             dataset = get_dataset(name)
-            writable_storage = dataset.get_writable_storage()
+            writable_storage = dataset.get_entity().get_writable_storage()
```

Comment:
How does `get_entity` work? Since there will be more than one entity associated with a dataset, should this method actually exist at all? It seems in this particular case, we'd be better served by just iterating over all of the writable storages.

Reply:
Indeed, there is no reason to rely on the default entity for this. Just iterate over all of them and get the writable storage from everyone.